Mathlib/LinearAlgebra/Span.lean

```diff
@@ -974,6 +974,16 @@ theorem ext_on_range {ι : Type _} {v : ι → M} {f g : M →ₛₗ[σ₁₂] M
 
 end AddCommMonoid
 
+section NoZeroDivisors
+
+variable (R M) [Ring R] [AddCommGroup M] [Module R M] [NoZeroSMulDivisors R M]
```

Comment:
```suggestion
variable (R M)
variable [Ring R] [AddCommGroup M] [Module R M] [NoZeroSMulDivisors R M]
```
I think your current spelling means there are now two different `R`s and `M`s that are active, because Lean 4's variable is weird.